scripts/chart/build

```diff
@@ -16,7 +16,7 @@ sed -i -e "s/%VERSION%/${CHART_VERSION}/g" build/chart/rancher/Chart.yaml
 sed -i -e "s/%APP_VERSION%/${APP_VERSION}/g" build/chart/rancher/Chart.yaml
 
 # get the value of shell-image, such as rancher/shell:v0.1.6, from the file pkg/settings/setting.go
-post_delete_base=$(grep -i shell-image pkg/settings/setting.go | cut -d "," -f 2 | sed -e 's/"//g' | sed -e 's/)//g' | sed -e 's/ //g') || ""
+post_delete_base=$(grep -i DefaultShellVersion pkg/buildconfig/constants.go | cut -d "=" -f 2 | sed -e 's/"//g' | sed -e 's/)//g' | sed -e 's/ //g') || ""
```

Comment:
this should pull in the environmental variable from sourcing export-config.

Reply:
Sure, that works better. I had originally just kept this Go source file parsing behavior.

Reply:
We should use the shell environmental varible since this use case is part of why we introduced the change in the first place.